backends/cuda-ref/ceed-cuda-ref-qfunctioncontext.c

```diff
@@ -351,14 +351,6 @@ static int CeedQFunctionContextGetData_Cuda(const CeedQFunctionContext ctx,
   return CEED_ERROR_SUCCESS;
 }
 
-//------------------------------------------------------------------------------
```

Comment:
Note - I deleted these for CUDA and HIP but intentionally left them in for `/cpu/self/ref` since that's our explainer backend for new backends